app/views/tasks/index.html.erb

```diff
@@ -1,50 +1,40 @@
-
-        </div>
-                <div class="card-body">
-                <%= task.description %>
-                <p class="card-text">Manger: <%= task.creator.name %></p>
-                <p class="card-text">Assigned to: <%= task.member.name %></p>
-                <%= link_to "View", task_path(task), class: "btn btn-outline-success" %>
-                <% if user_signed_in? %>
-                <%= link_to "Edit", edit_task_path(task), class: "btn btn-outline-info" %>
-                <%= button_to "Delete", task_path(task), method: :delete, data: {confirm: "Are you sure?"}, class: "btn btn-outline-danger" %>
-                <% end %>
-        </div>
-
-                <div class="card-footer">
-                <div class="form-check form-check-inline">
-                <%= radio_button_tag "status_#{task.id}", "todo", task.status == 'todo', class: "form-check-input", data: { task_id: task.id } %>
-                <label class="form-check-label btn btn-outline-primary" for="status_<%= task.id %>">Todo</label>
-                </div>
-                <div class="form-check form-check-inline">
-                <%= radio_button_tag "status_#{task.id}", "in-process", task.status == 'in-process', class: "form-check-input", data: { task_id: task.id } %>
-                <label class="form-check-label btn btn-outline-warning" for="status_<%= task.id %>">In Process</label>
-                </div>
-                <div class="form-check form-check-inline">
-                <%= radio_button_tag "status_#{task.id}", "done", task.status == 'done', class: "form-check-input", data: { task_id: task.id } %>
-                <label class="form-check-label btn btn-outline-success" for="status_<%= task.id %>">Done</label>
-                </div>
-                </div>
-            <div class="card-footer text-muted">
-                <small>Created <%= time_ago_in_words(task.created_at) %> ago, 
-                edited <%= time_ago_in_words(task.updated_at) %> ago</small>
-            </div>
+  <div class="container">
+    <div id="tasks">
+      <%= render @tasks %>
     </div>
+    <div class='dddd'>
     </div>
-<% end %>
-</div>
+    <div id="pagination" class='hidden'>
+      <%= will_paginate @tasks %>
+    </div>
+  </div>
 </div>
 
 
+<script>
+  <% if @tasks.next_page %>
+    var visitedUrl = undefined
+    $(window).on('scroll', function() {
+    // console.log($('.pagination a.next_page'));
+      var url = $('.pagination a.next_page').attr('href');
+      if (url && $(window).scrollTop() > $(document).height() - $(window).height() - 20) {
+        console.log('inside if scrolling');
+        console.log("url", url)
+        // $('.pagination').text('Loading more projects...');
+        if(visitedUrl !== url){
+            $.getScript(url);
+            visitedUrl = url
+        }
+      }
+    });
+  <% else %>
+    console.log('inside else');
+    $(window).off('scroll');
+    $('.pagination').remove();
+  <% end %>
+</script>
+
```

Comment:
As I above said you added this script in the application.js you do not need to add it here.